example/android/app/jni/Application.mk

```diff
@@ -4,8 +4,8 @@ APP_ABI := all
 APP_OPTIM := release
 APP_PLATFORM := android-8
 # GCC 4.8 Toolchain - requires NDK r9
-NDK_TOOLCHAIN_VERSION = 4.8
+NDK_TOOLCHAIN_VERSION = clang
```

Comment:
Last I tested the 4.8 GCC toolchain was missing `std::experimental:optional` which prompted the switch in our code.